src/grid/atomgrid.py

```diff
@@ -390,58 +389,26 @@ def _find_l_for_rad_list(radial_arrays, radius_list, degs):
         return degs[position]
```

Comment:
Can you please clarify this function a bit more?
1. The `Parameters` don't match the argument list; there is no r_sectors & radius should change to radius_list, ...
2. Update the function description (there is no r_sectors) and replace "magic L value" with more clear term.
3. Remove comments if not needed.